app/grandchallenge/algorithms/models.py

```diff
@@ -341,18 +341,21 @@ def assign_workstation_permissions(self):
                     perm=perm, user_or_group=group, obj=self.workstation
                 )
 
-    @cached_property
-    def latest_executable_image(self):
+    @property
```

Comment:
Why is this no longer cached? Would also need to update Methods and Workstations to be cached again.

Reply:
Accidentally changed this, sorry!